src/components/AdminDashboard/CompleteIncident.js

```diff
@@ -139,20 +139,30 @@ const CompleteIncident = props => {
 
             {/* Tags */}
             <div className="dropdown-text-wrap">
-              <p className="complete-incident-dropdown-titles-bold">Tags</p>
-              <div>
-                {incident.tags ? incident.tags.join(", ") : ''}
-              </div>
+              <p className="complete-incident-dropdown-titles-bold">Tags:</p>
+              <div>{incident.tags ? incident.tags.join(', ') : ''}</div>
+            </div>
+
+            {/* Confidence Rating: */}
```

Comment:
Since the confidence rating rates the Force Rank classification only, would it make more sense to put this inline with, or directly under Force Rank? And should we consider possibly relabeling it to something like "Force Rank Confidence" to provide clarity on what it's actually rating?

Reply:
Good point. I'm going to change the name to "Force Rank Confidence" so that the name is more descriptive and also move it just below the force rank.